tests/test_units.py

```diff
@@ -0,0 +1,99 @@
+"""Test the unit attribute of the Field class."""
+
+from corsikaio.subblocks.dtypes import Field
+from corsikaio.subblocks import get_units_from_fields
+
+from corsikaio.subblocks.run_header import run_header_fields
+from corsikaio.subblocks.event_header import event_header_fields
+from corsikaio.subblocks.data import (
+    particle_data_fields,
+    cherenkov_photons_fields,
+    mmcs_cherenkov_photons_fields,
+)
+from corsikaio.subblocks.longitudinal import (
+    longitudinal_data_fields,
+    longitudinal_header_fields,
+)
+from corsikaio.subblocks.event_end import event_end_fields
+from corsikaio.subblocks.run_end import run_end_fields
+
+
+def test_new_field():
+
+    field = Field(1, "test_field")
+    assert field.unit == None
```

Comment:
comparison to `None` should be `is None`.

Reply:
fixed